src/ethereum/cancun/vm/precompiled_contracts/point_evaluation.py

```diff
@@ -0,0 +1,76 @@
+"""
+Ethereum Virtual Machine (EVM) POINT EVALUATION PRECOMPILED CONTRACT
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Implementation of the POINT EVALUATION precompiled contract.
+"""
+from eth2spec.deneb.minimal import (
```

Comment:
I think you want to use the mainnet preset?
```suggestion
from eth2spec.deneb.mainnet import (
```

Reply:
updated